vlsi/example-vlsi-sky130

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+#
+# NOTE: the custom hooks are only used for sky130.
+
+import os
+
+import hammer_vlsi
+from hammer_vlsi import CLIDriver, HammerToolHookAction
+
+from typing import Dict, Callable, Optional, List
+
+from technology.sky130 import SKY130Tech
+
+def example_place_tap_cells(x: hammer_vlsi.HammerTool) -> bool:
+    if x.get_setting("vlsi.core.technology") == "sky130":
+        x.append('''
+# TODO
+# Place custom TCL here
+''')
+    return True
+
+def example_add_fillers(x: hammer_vlsi.HammerTool) -> bool:
+    if x.get_setting("vlsi.core.technology") == "sky130":
+        x.append('''
+# TODO
+# Place custom TCL here
+''')
+    return True
+
+def example_tool_settings(x: hammer_vlsi.HammerTool) -> bool:
+    if x.get_setting("vlsi.core.technology") == "sky130":
+        x.append('''
+# TODO
+# Place custom TCL here
+set_db route_design_bottom_routing_layer 2
+set_db route_design_top_routing_layer 5
+''')
+    return True
+
+def drc_blackbox_openram_srams(x: hammer_vlsi.HammerTool) -> bool:
+    if x.get_setting("vlsi.core.technology") == "sky130":
+        drc_box = ''
+        for name in SKY130Tech().openram_sram_names():
```

Comment:
Can both this DRC & LVS hook be made default in the Sky130 plugin itself? It appears you already know what the names are via `SKY130Tech().open_sram_names()`.

Reply:
moved them back to the tech plugin. don't remember what my reasoning for including those here was.